.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
```

Comment:
Can we split this PR up - make this PR just add CI, and then put the other stuff in a separate PR (to which CI will run against)